### ISSUE TITLE
migration error ALTER TABLE questions ADD COLUMN name_translations_jsonb jsonb DEFAULT '{}

### ISSUE THREAD

**Issue by bhaveshp3** [thecartercenter/elmo#551]
[migration error.txt](https://github.com/thecartercenter/elmo/files/2659888/migration.error.txt)

I am following the guideline given here ::
https://github.com/thecartercenter/elmo/blob/master/docs/production-setup.md

But getting error as attached in file. 
I am at v6.0 and executed command 

ubuntu@ip-172-31-41-127:~/elmo$ git checkout v6.0 
ubuntu@ip-172-31-41-127:~/elmo$ bundle install --without development test --deployment

**Comment by smoyth** [comment]
Version 6.0 is very old and we no longer use MySQL. Is there a reason you're attempting to install such an old version?

**Comment by bhaveshp3** [comment]
Thank you for your response.

Yes, I was doing the update of ELMO software from v5.0 to v9.0 as per guideline 
https://github.com/thecartercenter/elmo/blob/develop/docs/production-setup/xenial/production-setup.md
In the above document, it has been said to update to  v5.16.2 , but there is no nvm version for this version so I started with v6.0 
so during this process , error occurred.
But today , i reattempted the update process and this error is fixed automatically.

Thank you again.

**Comment by smoyth** [comment]
Ok I see! I didn't realize this was an upgrade process vs. a new install. Makes more sense now. I'm glad you got past it. Let us know if you have any further trouble. 5 -> 9 is a big upgrade! Congrats!

**Comment by bhaveshp3** [comment]
Yes, I have received during execution of following statement during upgrading to v6.11
**bundle exec rake db:migrate**
But I have ignored and same error inherited in all next upgrade to v9.0. I am not sure whether i have done correct or not. 
I am sure that It may impact Application functioning. But I could not find proper solutions. 
I am new to this Application and its environment. 
I would be thankful if any guide/help is provided to resolve this issue. 

Error is attached here.
![db_migrate_error](https://user-images.githubusercontent.com/16354213/49744977-fdb32b00-fcc3-11e8-8ad6-5dbf6dd3ff43.png)

Thank you!

**Comment by smoyth** [comment]
What version of Postgres are you using? Perhaps newer versions don't even allow those zero time values. You could try removing that migration file `db/migrate/20170411131240_fix_zero_time_values.rb` and see if things run cleanly. It may not cause problems for you.

**Comment by bhaveshp3** [comment]
@smoyth 
Sorry for late reply.
Thank you for valuable suggestion of removing the migration file. 
It worked for me and above errors are removed and db migrated successfully and running well so far.
Thank you again.

**Comment by smoyth** [comment]
Finally some good news today!
